recipes/draco/all/conanfile.py

```diff
@@ -0,0 +1,122 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class DracoConan(ConanFile):
+    name = "draco"
+    description = "Draco is a library for compressing and decompressing 3D " \
+                  "geometric meshes and point clouds. It is intended to " \
+                  "improve the storage and transmission of 3D graphics."
+    license = "Apache-2.0"
+    topics = ("conan", "draco", "3d", "graphics", "mesh", "compression", "decompression")
+    homepage = "https://google.github.io/draco/"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
```suggestion
    settings = "os", "arch", "compiler", "build_type"
    short_paths = True
```
https://docs.conan.io/en/latest/reference/conanfile/attributes.html#short-paths